src/request/sign-transaction/SignTransaction.css

```diff
@@ -1,154 +1,300 @@
+    margin-top: 2rem;
 }
 
-.transaction .total .nim-symbol {
+#account-details .total .balance {
+    font-size: 3rem;
+}
+
+#account-details .total .nim-symbol {
     margin-left: 1rem;
+    font-weight: 800;
 }
 
-.transaction .fee-section {
-    text-align: center;
+#account-details .wallet-label {
+    margin-top: 1.5rem;
+}
+
+#account-details .address {
+    width: 100%;
+    max-width: calc(28.25 * var(--nimiq-size, 8px));
+    font-family: 'Fira Mono', monospace;
     opacity: 0.5;
+    margin-top: 7rem;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    font-size: 3rem;
 }
 
-.transaction .data-section {
-    margin: 2rem 3rem 0;
+.chunk {
+    margin: calc(0.875 * var(--nimiq-size, 8px)) 0;
+    line-height: 1.11;
+    width: 33%;
+    height: 100%;
     text-align: center;
+    box-sizing: border-box;
+    text-transform: uppercase;
 }
 
-.layout-standard .nq-card-footer {
-    border-radius: 0;
+.space {
+    font-size: 0;
 }
 
-.page-footer {
-    padding: 1rem;
+.transaction .nq-card-body {
+    background-color: white;
 }
 
-.layout-standard ~ .page-footer {
-    background: white;
-    border-bottom-left-radius: 1rem;
-    border-bottom-right-radius: 1rem;
+.transaction .accounts {
+    flex-direction: row;
+    align-items: flex-start;
+    padding-bottom: 4rem;
 }
 
-/*
-#######################################
-#### C H E C K O U T   L A Y O U T ####
-#######################################
-*/
+.transaction .account .total,
+.transaction .account .address {
+    display: none;
+}
 
-.payment-info-line {
-    margin-top: -3.5rem;
+.transaction > .nq-card-body:after {
+    content: ' ';
+    display: flex;
+    height: 0px;
+    border-top: 1px solid #1F2348;
+    opacity: 0.1;
 }
 
-.transaction-section {
-    background: var(--nimiq-card-header-bg);
-    border-bottom: 1px solid var(--nimiq-card-border-color);
-    padding-bottom: 3rem;
-    border-top-left-radius: 1rem;
-    border-top-right-radius: 1rem;
-    flex-grow: 1;
+.transaction .account {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-decoration: none;
+    cursor: pointer;
+    color: var(--nimiq-dark-blue);
 }
 
-.layout-checkout h1 {
-    font-size: 3rem;
-    line-height: 3.625rem;
-    letter-spacing: 0.021em;
-    font-weight: normal;
-    padding: 4rem 4rem 1rem 4rem;
-    margin: 0;
+.transaction .account .identicon {
+    width: 10rem;
+    height: 10rem;
+    transition: transform 0.45s ease;
 }
 
-.layout-checkout h1 strong {
-    font-weight: bold;
+.transaction .account:focus .identicon,
+.transaction .account:hover .identicon {
+    transform: scale(1.1);
 }
 
-.layout-checkout h1 .nim-symbol::before {
-    opacity: 1;
+.transaction .label {
+    font-size: 2rem;
+    margin: 1.25rem 0;
 }
 
-.layout-checkout .account {
-    flex-direction: row;
-    padding: 1.5rem 4rem;
+.transaction .wallet-label {
+    font-size: 1.25rem;
+    margin-top: 0;
+    display: none;
```

Comment:
Already has class `display-none` that is dynamically removed when the wallet label is known.
This line here however prevents that it is ever visible.